CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
 - Fix: -noCache now avoids populating cache on create/update subscription operations (#2147)
 - Deprecated: /ngsi10 and /ngsi9 as URL path prefixes
 - Fix: potential bug in internal logic at NGSIv2 GET subscription operations, possibly provoking crashes
+- Add: NGSIv2 notification templates (Issue #2015)
```

Comment:
At the end, taking into account how documentation and API artifacts (e.g. httpCustom) has evolved,I think that it is better to talk about "custom notifications", i.e.

"Add: NGSIv2 custom notifications (Issue #2015)"